.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.nosetestsEnabled": false,
+    "python.testing.pytestEnabled": true,
```

Comment:
*For future reference, we'll be adding any file inside of the .vscode folder into a .gitignore. Files that are specific to your development environment such as vscode settings should be kept out of github just in case it'll conflict with other people's settings.